src/index.html

```diff
@@ -7,6 +7,308 @@
+          class="card__button"
+          data-qa="card-hover">
+          Купить
+        </a>
+      </div>
+
+      <div class="card card--location" data-qa="card">
+        <img src="images/imac.jpeg"
+          alt="imac"
+          class="card__photo">
+
+        <h3 class="card__name">
+          Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+        </h3>
+        <p class="card__code">Код товара: 195434</p>
+
+        <div class="card__rating">
+          <div class="card__stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__reviews">Отзывов: 5</p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price__word">Цена:</p>
+          <p class="card__price__value">69 999 грн</p>
+        </div>
+
+        <a href="#"
+          class="card__button"
+          data-qa="card-hover">
+          Купить
+        </a>
+      </div>
+
+      <div class="card card--location" data-qa="card">
+        <img src="images/imac.jpeg"
+          alt="imac"
+          class="card__photo">
+
+        <h3 class="card__name">
+          Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+        </h3>
+        <p class="card__code">Код товара: 195434</p>
+
+        <div class="card__rating">
+          <div class="card__stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__reviews">Отзывов: 5</p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price__word">Цена:</p>
+          <p class="card__price__value">69 999 грн</p>
+        </div>
+
+        <a href="#"
+          class="card__button"
+          data-qa="card-hover">
+          Купить
+        </a>
+      </div>
+
+      <div class="card card--location" data-qa="card">
+        <img src="images/imac.jpeg"
+          alt="imac"
+          class="card__photo">
+
+        <h3 class="card__name">
+          Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+        </h3>
+        <p class="card__code">Код товара: 195434</p>
+
+        <div class="card__rating">
+          <div class="card__stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__reviews">Отзывов: 5</p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price__word">Цена:</p>
+          <p class="card__price__value">69 999 грн</p>
+        </div>
+
+        <a href="#"
```

Comment:
Use `<button>` instead of `<a>` if you don't want to e.g. redirect to another url. Rememeber to add one of `type` attribute:
```html
<button type="button|submit|reset">
```

More info about [button](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button) tag

Reply:
Each attribute should be on a new line and indented with 2 spaces

```suggestion
        <a 
          href="#"
```